src/IonDecimal.ts

```diff
@@ -12,8 +12,9 @@
  * language governing permissions and limitations under the License.
  */
 
-import {LongInt} from "./IonLongInt";
+import JSBI from 'jsbi';
```

Comment:
nit: be consistent on the use of single VS double quotes

Reply:
Ah, sure. I happen to like single quotes for imports, but I'll switch them for consistency's sake.